pydevd_plugins/extensions/pydevd_plugin_omegaconf.py

```diff
@@ -0,0 +1,114 @@
+# based on https://github.com/fabioz/PyDev.Debugger/tree/main/pydevd_plugins/extensions
+import os
+import sys
+from functools import lru_cache
+from typing import Any, Dict, Sequence
+
+from _pydevd_bundle.pydevd_extension_api import (  # type: ignore
+    StrPresentationProvider,
+    TypeResolveProvider,
+)
+
+
+@lru_cache(maxsize=128)
+def find_mod_attr(mod_name: str, attr: str) -> Any:
+    mod = sys.modules.get(mod_name)
+    return getattr(mod, attr, None)
+
+
+class OmegaConfDeveloperResolver(object):
+    def can_provide(self, type_object: Any, type_name: str) -> bool:
+        Node = find_mod_attr("omegaconf", "Node")
+        return Node is not None and issubclass(type_object, Node)
```

Comment:
Why check for `None`? (other places where `find_mod_attr()` is used don't seem to do such check)

Reply:
To be safe: if the extension ends up running without OmegaConf installed, I don't want it to break the debugger.
Once we decided that we can support an object, it means OmegaConf must be available.
In addition, further interactions are limited to a specific object.